include/dblayer/mysqli_innodb.php

```diff
@@ -26,10 +26,10 @@ class DBLayer
 
 	var $datatype_transformations = array(
 		'%^SERIAL$%'	=>	'INT(10) UNSIGNED AUTO_INCREMENT'
-	);
-
-
-	function DBLayer($db_host, $db_username, $db_password, $db_name, $db_prefix, $p_connect)
+	)
+	
+	
+	function __construct($db_host, $db_username, $db_password, $db_name, $db_prefix, $p_connect)
```

Comment:
Sorry about this, Notepad++ is acting crazy. I tried PHPStorm a bit, but I found it rather complicated...